index.html

```diff
@@ -0,0 +1,217 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
```

Comment:
<img width="330" alt="whaaaa" src="https://user-images.githubusercontent.com/77843050/205431188-a97cdbed-9775-4f5f-808e-1f3374293ac2.png">

- [ ] Do we see the same thing? 👀... this is what we are welcome with when we visit your page, please make sure you are well importing your CSS file in the HTML and resubmit your code with the correct design.

Reply:
The design is working well on my local. I have no idea why it looks that way on your local.